cli.ts

```diff
@@ -415,12 +415,14 @@ const scaffoldProject = (i18n: i18n.t) =>
 			await eager(rm(gitDir));
 			log('    ✓ Remove Git directory');
 
-			log('    ✓ Scan for plugins');
-
-			await eager(exec('npm run setup 2>&1 > /dev/null', {}, false, destDir));
-			log('    ✓ Install dependencies');
+			await eager(exec('npm install 2>&1 > /dev/null', {}, false, destDir));
```

Comment:
@hu3man should we just suppress the stdout? Otherwise if something fails due to a network error than it won't be apparent to the end-developer or the Taqueria team when troubleshooting. Thoughts?

Reply:
That's probably the better option @mweichert 

That would just be doing the following right?
```
'npm run setup > /dev/null`
```

Reply:
@hu3man yep, I think that would be better.